src/Akeneo/Pim/Enrichment/Product/back/API/Event/Completeness/ChangedProductCompleteness.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Enrichment\Product\API\Event\Completeness;
+
+/**
+ * @copyright 2022 Akeneo SAS (http://www.akeneo.com)
+ * @license http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class ChangedProductCompleteness
```

Comment:
```suggestion
final class ProductCompletenessDecreased
```
and `ProductCompletenessIncreased` ? 


"changed" is generally a generic naming. Everything change when you manipulate data.

Reply:
Is there a functional need to know if the completeness increased or decreased? The classes would be the same, so if no special needs, it could complexify something that is already not-easy to read.

Reply:
https://verraes.net/2019/05/patterns-for-decoupling-distsys-fat-event/

> Adapting the API of a producer to fit the needs of the consumer, should always be done with care. With lots of consumers that each have their own specific demands, it could quickly become mess. Events become huge, and as consumers come and go, it becomes unclear wich attributes are still relevant.

Reply:
ProductWasCompleted